<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PinWidget</class>
 <widget class="QWidget" name="PinWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="stepList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="addStep">
       <property name="text">
        <string>Pin current Step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="showStep">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Display Step</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showCell">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Display Cell</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="repeatStep">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Repeat Step</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Offset:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QDoubleSpinBox" name="xOffset">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-999.990000000000009</double>
         </property>
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="yOffset">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="zOffset">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="multLabel">
       <property name="text">
        <string>Cell multiply:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="multLayout">
       <item>
        <widget class="QSpinBox" name="xMultBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="yMultBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="zMultBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cell vector fitting:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="xFit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="yFit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="zFit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>z</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="insertStep">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Insert pinned into current Step</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="delStep">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Delete pinned Step</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>setMult(int)</slot>
  <slot>setOffset(double)</slot>
 </slots>
</ui>
